001-cron.php

```diff
@@ -111,6 +111,16 @@ function wpcom_vip_cron_control_event_object_to_string( $event ) {
 	return sprintf( 'ID: %1$d | timestamp: %2$s | action: %3$s | action_hashed: %4$s | instance: %5$s | home: %6$s', $event->ID, $event->timestamp, $event->action, $event->action_hashed, $event->instance, home_url( '/' ) );
 }
 
+/**
+ * Callback for 'a8c_cron_control_uncacheable_cron_option' action. Send an alert to IRC and Slack in case of cron option being too big.
+ *
+ * @param $event object
+ */
+function wpcom_vip_log_cron_control_uncacheable_cron_option( $option_size, $buckets, $option_flat_count ) {
+	$message = sprintf( 'Cron Control Cron Option Uncacheable Alert - home: %s | option size: %d | buckets: %d | option flat count: %d', home_url( '/' ), $option_size, $buckets, $option_flat_count );
+	wpcom_vip_irc( 'vipv2-alerts', $message, 2, '', 5 );
```

Comment:
Should we have a value for the 4th parameter here?

Reply:
Instead of 1 message every 5 seconds, which would still be a _lot_ of alert messages going off, we should dial it back to about once every 15 minutes.